src/components/Header/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import logoImg from '../../assets/logo.svg';
+import { UserContext } from '../../contexts/UserContext';
+import Profile from '../Profile/Profile';
+import { Link, useLocation } from 'react-router-dom';
+import * as S from './Header.styled';
+
+export default function Header() {
+  const { user } = useContext(UserContext);
+  const location = useLocation();
+
+  const $isSticky = location.pathname !== '/folder';
```

Comment:
`location.pathname !== '/folder'` 이 부분을 의미하는 게 isSticky 는 아닐테니 변수명을 수정하는걸 추천합니다
그 변수가 어디서 활용되는지보다는 변수 그 자체의 이름을 담고 있어야 나중에도 헷갈리지 않아요
아래에서는 변수명 작성한거 넘겨주면 되니까요

```suggestion
  <S.Header $isSticky={isXX}>
```